src/client/StreamOpts.ts

```diff
@@ -5,7 +5,7 @@ export interface StreamOpts {
     bitrateKbps?: number;
     maxBitrateKbps?: number;
     hardware_acceleration?: boolean;
-    video_codec?: 'H264' | 'VP8';
+    video_codec?: string;
```

Comment:
Should we just make a type for the supported video codecs? , i.e 
```typescript
export type SupportedVideoCodec = "H264" | "H265" | "VP8" | "VP9" | "AV1";
```

Reply:
It's possible, let me think about it.

Reply:
Any updates? We could just leave it as is.

Reply:
Yea we can probably leave it for later, I'm pretty busy with university assignments